docs/source/release_notes.rst

```diff
@@ -10,6 +10,8 @@ Release Notes
         * Added utility method to create list of components from a list of ``DataCheckAction`` :pr:`1907`
         * Updated ``validate`` method to include a ``action`` key in returned dictionary for all ``DataCheck``and ``DataChecks`` :pr:`1916`
         * Aggregating the shap values for predictions that we know the provenance of, e.g. OHE, text, and date-time. :pr:`1901`
+        * Added ``score_batch`` and ``train_batch`` methods to ``AutoMLSearch`` :pr:`1913`
```

Comment:
think this should be `train_pipelines` and `score_pipelines`

Reply:
You're totally right 🙈